src/types/types.ts

```diff
@@ -2,6 +2,7 @@ export type DeployTokenRequest = {
   name: string;
   symbol: string;
   decimals: number;
+  nftAddress: `0x${string}` | null;
```

Comment:
If we dont fill the input it would be "undefined", not "null" i think?
Maybe just mark it as optional?